tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -39,6 +39,7 @@ services:
       - "../../docker-compose/_sources/local_settings.py:/etc/tower/conf.d/local_settings.py"
       - "../../docker-compose/_sources/SECRET_KEY:/etc/tower/SECRET_KEY"
       - "../../docker-compose/_sources/receptor/receptor-awx-{{ loop.index }}.conf:/etc/receptor/receptor.conf"
+      - "../../docker-compose/_sources/receptor/receptor-awx-{{ loop.index }}.conf.lock:/etc/receptor/receptor.conf.lock"
```

Comment:
This feels wrong to me. Give me until tomorrow to play around with another option.

Reply:
yah this feels wrong...if i want to fix it i would create multiple dirs under _source and each will be for a different receptor node this way i can just mount the whole dir

Reply:
we can address it in follow up PR